container/shim/src/index.js

```diff
@@ -91,9 +95,10 @@ if (cluster.isPrimary) {
     })
 
     if (req.headers.range) {
-      let [start, end] = req.headers.range.split('=')[1].split('-')
-      start = parseInt(start, 10)
-      end = parseInt(end, 10)
+      const range = req.headers.range.split('=')?[1]?.split('-')
+      assert(range, 'TODO')
```

Comment:
Do we want to fail on invalid range? Or return empty bytes?

Reply:
I think a 4xx response is appropriate, so the client can know they need to adjust their request

Reply:
Then we need either try catch or do a truthy check returning 400